sources/web/datalab/server.ts

```diff
@@ -153,6 +153,12 @@ function handleRequest(request: http.ServerRequest,
     return;
   }
 
+  if (requestPath.indexOf('/api/basepath') === 0) {
+    response.statusCode = 200;
+    response.end(appSettings.datalabBasePath);
```

Comment:
Since I'm not sure, what exact format is this returning?  is it pure text or is it a JSON dict?